test/pxutils.test.js

```diff
@@ -32,4 +32,14 @@ describe('PX Utils - pxutils.js', () => {
         formattedHeaders[0]['value'].should.be.exactly('v');
         return done();
     });
+
+    it('should extract cookie names from the cookie header', (done) => {
+        var cookieHeader = '_px3=px3Cookie;tempCookie=CookieTemp; _px7=NotARealCookie';
+        var formattedHeaders = pxutil.extractCookieNames(cookieHeader);
+        (Object.prototype.toString.call(formattedHeaders)).should.be.exactly('[object Array]');
```

Comment:
why not `formattedHeaders.toString()`? this will actually pretty print the array instead of the general `[object Array]`

Reply:
Because  I want to know the type, not the value

Reply:
In that case
```suggestion
        Array.isArray(formattedHeaders).should.be.true()
```